kyuubi-server/src/test/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilderSuite.scala

```diff
@@ -79,7 +80,7 @@ class FlinkProcessBuilderSuite extends KyuubiFunSuite {
 
   private val javaPath = s"${envDefault("JAVA_HOME")}${File.separator}bin${File.separator}java"
   private val flinkSqlClientJarPathSuffix = s"${File.separator}opt${File.separator}" +
-    s"flink-sql-client_$SCALA_COMPILE_VERSION-$FLINK_COMPILE_VERSION.jar"
+    s"flink-sql-client-$FLINK_COMPILE_VERSION.jar"
```

Comment:
As Kyuubi claims to support both Flink 1.14 and 1.15, we should support both of them.